build.gradle

```diff
@@ -26,28 +26,27 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    def embulkVersion = "0.10.36"
-    compileOnly "org.embulk:embulk-api:${embulkVersion}"
+    def embulkVersion = "0.11"
```

Comment:
This plugin does not support embulk v0.11 yet.

Reply:
Hi! I tested it with with embulk 0.11 and it seemed to work fine;)

Reply:
ok. Could you fix the tests & CI?

Reply:
Done! (Sorry, took some time to do this)